src/main/java/ladder/view/InputView.java

```diff
@@ -19,13 +19,48 @@ public int readLadderHeight() {
         System.out.println();
         System.out.println("최대 사다리 높이는 몇 개인가요?");
 
-        String input = scanner.nextLine();
+        String input = readLine();
 
         return parseToInt(input);
     }
 
-    private List<String> parseStringToList(String input) {
-        return Arrays.stream(input.split(",", -1)).toList();
+    public List<String> readPrizeNames(int playerCount) {
+        System.out.println();
+        System.out.println("실행 결과를 입력하세요. (결과는 쉼표(,)로 구분하세요)");
+
+        String input = readLine();
+        List<String> prizeNames = parseStringToList(input);
+
+        validatePrizeSize(playerCount, prizeNames.size());
+
+        return prizeNames;
+    }
+
+    private void validatePrizeSize(int playerCount, int prizeCount) {
+        if (playerCount != prizeCount) {
+            throw new IllegalArgumentException("참가자 수와 결과 수가 일치해야 합니다.");
```

Comment:
가능하면 예외문은 풍부하게 맥락을 담아주면 좋습니다. 
```suggestion
            throw new IllegalArgumentException("참가자 수와 결과 수가 일치해야 합니다. 참가자 수 : " + playerCount + ", 결과 수 : " + prizeCount);
```

Reply:
오.. 이걸 생각 못했네요